packages/connectivity_plus/connectivity_plus/README.md

```diff
@@ -84,16 +82,13 @@ dispose() {
 
 ### Android
 
-Connectivity changes are no longer communicated to Android apps in the background starting with Android O (8.0). You should always check for connectivity status when your app is resumed._ The broadcast is only useful when your application is in the foreground.
+Connectivity changes are no longer communicated to Android apps in the background starting with Android O (8.0). You should always check for connectivity status when your app is resumed. The broadcast is only useful when your application is in the foreground.
 
 ### iOS
 
-On iOS, the connectivity status might not update when WiFi status changes, this is a known issue that affects mainly simulators.
-
-Starting on iOS 12, the implementation uses `NWPathMonitor` to obtain the enabled connectivity types.
-
-We noticed that this observer can give multiple or unreliable results. For example, reporting connectivity "none" followed by connectivity "wifi" right after reconnecting.
+On iOS simulators, the connectivity types stream might not update when Wi-Fi status changes. This is a known issue.
```

Comment:
Changed to simulators only because I believe that this note was created before `NWPathMonitor` usage, which brings results better, even though with some issues noted below. 

Considering that iOS 12 is the minimum version now rephrased to only mention simulators as can only reproduce the issue on simulators only with plugin version 6.0.1.